### PR TITLE
TLS upate closes  #8454

### LINKED ISSUE #8454: TLS 1.3 is now latest standard
Should correct that TLS 1.3 is now the newest released standard. 

https://datatracker.ietf.org/doc/rfc8446/

Any idea when MSFT products will start supporting it in their products including AD/ADLDS, etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7341fa0a-f474-6c2d-d3c0-7e2df12fbb65
* Version Independent ID: e99c489a-7518-d293-ad93-7eaca8b7191a
* Content: [Transport Layer Security (TLS) best practices with the .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls)
* Content Source: [docs/framework/network-programming/tls.md](https://github.com/dotnet/docs/blob/master/docs/framework/network-programming/tls.md)
* Product: **dotnet-framework**
* GitHub Login: @blowdart
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by dagolinuxoid**
Some minor updates regarding transport layer security, since TLS 1.3 protocol has been published.
Fixes #8454

### CODE REVIEW

File: docs/framework/network-programming/tls.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ author: "blowdart"
 ---
 # Transport Layer Security (TLS) best practices with the .NET Framework
 
-The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.2](https://tools.ietf.org/html/rfc5246) is the newest released standard and provides security improvements over previous versions. TLS 1.2 will eventually be replaced by [TLS 1.3](https://tools.ietf.org/html/draft-ietf-tls-tls13-22). This article presents recommendations to secure .NET Framework applications that use the TLS protocol.
+The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.2](https://tools.ietf.org/html/rfc5246) is a standard that provides security improvements over previous versions. TLS 1.2 will be replaced by the newest released standard [TLS 1.3](https://tools.ietf.org/html/draft-ietf-tls-tls13-22) which is faster and has improved security. This article presents recommendations to secure .NET Framework applications that use the TLS 1.2 protocol.
```
Comment (blowdart) [inline docs/framework/network-programming/tls.md]: TLS 1.2 will be replaced > TLS 1.2 will eventually be replaced

**Review by karelz — CHANGES_REQUESTED** [review]
+1 on changing the wording as suggested above. Otherwise LGTM.

**Comment by karelz** [comment]
@dagolinuxoid thanks for fast turnaround on putting up the PR!

File: docs/framework/network-programming/tls.md (hunk 1 above)
Comment (dagolinuxoid) [reply docs/framework/network-programming/tls.md]: added

File: docs/framework/network-programming/tls.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ author: "blowdart"
 ---
 # Transport Layer Security (TLS) best practices with the .NET Framework
 
-The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.2](https://tools.ietf.org/html/rfc5246) is the newest released standard and provides security improvements over previous versions. TLS 1.2 will eventually be replaced by [TLS 1.3](https://tools.ietf.org/html/draft-ietf-tls-tls13-22). This article presents recommendations to secure .NET Framework applications that use the TLS protocol.
+The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.2](https://tools.ietf.org/html/rfc5246) is a standard that provides security improvements over previous versions. TLS 1.2 will eventually be replaced by the newest released standard [TLS 1.3](https://tools.ietf.org/html/draft-ietf-tls-tls13-22) which is faster and has improved security. This article presents recommendations to secure .NET Framework applications that use the TLS 1.2 protocol.
```
Comment (karelz) [inline docs/framework/network-programming/tls.md]: Change TLS 1.3 link to final version: https://tools.ietf.org/html/rfc8446

**Review by karelz — CHANGES_REQUESTED** [review]

File: docs/framework/network-programming/tls.md (hunk 2 above)
Comment (dagolinuxoid) [reply docs/framework/network-programming/tls.md]: @karelz my bad. I hope it's a last update :).

**Review by karelz — APPROVED** [review]

File: docs/framework/network-programming/tls.md (hunk 2 above)
Comment (karelz) [reply docs/framework/network-programming/tls.md]: Thanks @dagolinuxoid!

**Pull request merged**